tox.ini

```diff
@@ -64,8 +62,10 @@ sitepackages = true
 # f'' is used inside docs
 basepython = python3
 deps =
-    flake8>=3.6.0,<4
-    yamllint>=1.15.0,<2
+    pre-commit>=1.15.2
+commands =
+    # to run a single linter you can do "pre-commit run flake8"
+    python -m pre_commit run {posargs:--all}
```

Comment:
@ssbarnea please add this as well: https://github.com/tox-dev/tox/blob/a338734/tox.ini#L61